src/components/CompareResults/SubtestsResults/SubtestsResultsTable.tsx

```diff
@@ -134,7 +134,7 @@ function resultMatchesColumnFilter(
   const cellConfiguration = cellsConfiguration.find(
     (cell) => cell.key === columnId,
   );
-  if (!cellConfiguration || !cellConfiguration.filter) {
+  if (!cellConfiguration || !('filter' in cellConfiguration)) {
```

Comment:
This is for Typescript. Type refining works with the `in` operator but not with the form we were gusin before, curiously.  I asked them and it seems to be made on purpose (see https://github.com/microsoft/TypeScript/issues/51181#issuecomment-2517826823).